src/main/java/mods/eln/Eln.java

```diff
@@ -6905,12 +6905,12 @@ private void recipeMacerator() {
         maceratorRecipes.addRecipe(new Recipe(findItemStack("Tungsten Ingot"),
             new ItemStack[]{findItemStack("Tungsten Dust", 1)}, 0.5 * f));
 
-        maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.cobblestone),
-            new ItemStack[]{new ItemStack(Blocks.gravel)}, 1.0 * f));
+        maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.cobblestone,32),
+            new ItemStack[]{new ItemStack(Blocks.gravel)}, 10.0 * f));
         maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.gravel),
-            new ItemStack[]{new ItemStack(Items.flint)}, 1.0 * f));
-        maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.dirt),
-            new ItemStack[]{new ItemStack(Blocks.sand)}, 1.0 * f));
+            new ItemStack[]{new ItemStack(Items.flint)}, 0.25 * f));
+        maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.stone),
+            new ItemStack[]{new ItemStack(Blocks.sand)}, 5.0 * f));
```

Comment:
I think it'd be better to use the mod-typical chain: Stone to cobble to sand, or alternately stone to cobble to gravel to sand.

As for earth, it would actually make physical sense to allow macerating earth to clay. Well, some physical sense. More than having any other block as input to that.